package.json

```diff
@@ -29,7 +30,7 @@
     "build:clean": "rimraf lib && rimraf dist",
     "build:dist": "rollup -c && rollup -c --min",
     "build:lib": "babel src -d lib",
-    "build:flow": "flow-copy-source --verbose src lib",
+    "build:flow": "echo \"//@flow\n\nexport * from '../src';\" > lib/index.js.flow",
```

Comment:
super interesting!

@jamiebuilds what do you think?

Reply:
Ha. Thats an interesting one. Nice

Reply:
Leaving this as reference - https://blog.kentcdodds.com/distributing-flow-type-definitions-for-node-and-browser-modules-3952ad38b357